### PR TITLE
Create ml group

### PR DESCRIPTION
**Pull request by ljgarcia**
This PR:
* Add the machine learning page
* Add pages for ML leads Fotis and Dan
* Fix bug in people layout that prevented the rendering of groups for which the person is lead rather than member

### CODE REVIEW

**Comment by ljgarcia** [comment]
@fpsom please have a look to the text for the ML group and let me know of any suggestions, thanks

File: pages/_people/FotisPsomopoulos.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: person-details
+id: LeylaJaelCastro
```
Comment (fpsom) [inline pages/_people/FotisPsomopoulos.md:3]: maybe change the id to fpsom?

Comment (ljgarcia) [reply pages/_people/FotisPsomopoulos.md]: Done

**Comment by ljgarcia** [comment]
@fpsom @ivanmicetic @albangaignard this PR is ready for review, thanks

**Review by gtsueng — APPROVED** [review]
Did not find any issues

**Pull request merged**